vars/starter.yml

```diff
@@ -0,0 +1,24 @@
+---
+drupal_build_composer_project: true
+drupal_composer_dependencies: []
+# Presently pointing at the main branch.
+drupal_composer_project_package: 'islandora/islandora-starter-site:dev-main'
+
+# XXX: Strictly, irrelevant due to using the `--existing-config` flag.
+drupal_install_profile: minimal
+
+# None necessary, due to using an existing configuration.
+drupal_enable_module: []
```

Comment:
Same deal: Strictly, doesn't need to be specified here (the default in the `geerlingguy.drupal` is an empty list); however, thought to leave it to maintain something of a parallel structure with the `demo` and `standard` "profiles"... could be useful for people looking to hack on things?

Reply:
Adjusted, to leave with a comment and with it commented out.